src/components/Editor/CodeEditor.tsx

```diff
@@ -81,12 +82,12 @@ export default function CodeEditor(props: CodeEditorProps) {
         const newPeerClientsId: string[] = event.value[doc.getKey().toIDString()];
         const setNewPeerClientsId = new Set(newPeerClientsId);
 
-        for (const clientId of Object.keys(peerClients)) {
+        Object.keys(peerClients).forEach((clientId) => {
```

Comment:
Can't we use the `for of` pattern?

Reply:
Currently, lint is strict, preventing the use of `for of`

Would it be better to change it to ignore this part?

Reply:
I mean a change to be able to use "for of"

Reply:
Can you further explain which lint error occurred?

Reply:
@hackerwins 

The lint error is as follows. Airbnb lint does not allow it.
```
iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations. eslint no-restricted-syntax
```

https://eslint.org/docs/rules/no-restricted-syntax

airbnb issue
https://github.com/airbnb/javascript/issues/1271

Reply:
I have a similar idea to [this comment](https://github.com/airbnb/javascript/issues/1271#issuecomment-281254652). I think the rule is opinionated and `for of` is easier to read than `.forEach()`.

How about we use `for of` in this project?

Reply:
I also like to use `for of` 
It seems that it can be flexibly in many places